model-optimizer/extensions/ops/interpolate.py

```diff
@@ -59,6 +60,8 @@ def infer_for_opset4(node: Node):
         for i, axis in enumerate(axes):
             output_shape[axis] = math.floor(scales[i] * output_shape[axis] + 1.0e-5)
 
+    # PermuteInputs().set_input_permutation(node.in_node(3), node, 'output:0', 'shape')
```

Comment:
Please, remove commented unneeded code

Reply:
This line should not be commented. For the TF models (or any other model with NHWC layout) this permutation is required.

Reply:
Fixed and 'shape' was replaced by 'axis'.